x-pack/legacy/plugins/snapshot_restore/public/app/sections/home/policy_list/policy_details/tabs/tab_summary.tsx

```diff
@@ -80,7 +86,7 @@ export const TabSummary: React.FunctionComponent<Props> = ({ policy }) => {
   const fullIndicesList =
     indices && indices.length && indices.length > 10 ? (
       <ul>
-        {indices.map((index: string) => (
+        {(Array.isArray(indices) ? indices : indices.split(',')).map((index: string) => (
```

Comment:
We normalize `indices` to an array here and on line 56. Would it be worth extracting this into a common variable?

```js
const indicesArray = Array.isArray(indices) ? indices : indices.split(',');
```